CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+
+**Make sure that the _base_ repository is set to "/SplitScreen-Me/splitscreenme-nucleus"!** 
+
+Otherwise, you will be attempting to merge your changes with a different repository. Double-check that you will be merging with the correct head branch as well (generally "master") Also verify that the _head_ repository is set to your fork and the branch of your fork where you made all the changes that you would like to implement.
+
+It is a good idea to create a detailed (but concise) comment to accompany the initial pull request to explain:
+* The issue the pull request is addressing
+* What is changed
+* Testing procedures used and their results
+* Potential issues or challenges encountered
+* Any questions or other comments for contribution reviewers
+
+#### Review Process
+All pull requests will approved by a sufficient number of project maintainers (those with write access to the project repository) before any changes are merged. Before aproving changes, reviewers will determine whether:
+* Changes do not interfere with existing features or design plans
+* There are overlap issues (i.e. other developers are already assigned to it) 
+* Sufficient testing occurred
+* Quality standards are upheld
+
+While it is certainly _possible_ that a quality pull request might be accepted right away, it should be expected that there will be some changes requested of the pull request, and issues that need to be discussed and resolved before it is approved. The pull request post itself is a great place to have these conversations, though contributors should not be afraid to directly message fellow developers either, when doing so is helpful to resolving any issues.
+
+### Implementing Changes
+
+#### Approval
+Once the review process is completed to the satisfaction of project enough project maintainers, a project maintainer will merge the pull request with the "master" branch, incorporating any changes within the pull request.
+
+At this point, the contributor's job is done! If your pull request made it this far, congratulations! Your contribution is now part of the project!
+
+#### Release Builds
+Periodically, the current lead developer(s) of Nucleus Coop will decide when enough changes have been incorporated into the master branch to justify a new release. After sufficient testing by project maintainers to ensure build stability, a release will be created and published alongside a change log of implemented changed as improvements. Releases include source code along with a more convenient installer and a "portable" (pre-compiled zip folder) option for users to run the program.
+
+Currently, there is no set release schedule, and releases will be published when ready and verified stable.
+
+### Code Standards
+* When in doubt, structure your code to err on the side of readability!

```

Comment:
As a non-native English speaker, I had trouble understanding what err meant in this sentence. Perhaps it should be re-worded? Feel free to disagree
```suggestion
* When in doubt, structure your code to prioritize readability!
```